src/pyfaf/config.py

```diff
@@ -87,6 +90,24 @@ def load_config():
 
     return result
 
+def load_log_config():
+    """
+    Load main config file for logging.
+    """
+    main_log_config_file = None
+    fpath = os.path.join(MAIN_CONFIG_DIR, MAIN_LOG_CONFIG_FILE)
+
+    if CONFIG_LOG_FILE_ENV_VAR in os.environ:
+        fpath = os.environ[CONFIG_LOG_FILE_ENV_VAR]
+
+    if os.access(fpath, os.R_OK):
```

Comment:
One thing is that these checks are prone to TOCTOU bugs. Even https://docs.python.org/3.7/library/os.html#os.access provides a better alternative.